.github/workflows/deploy-my-kibana.yml

```diff
@@ -20,11 +20,12 @@ permissions:
 
 jobs:
   deploy-my-kibana:
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/oblt-deploy'}}
+    if: ${{ github.event.issue.pull_request && (github.event.comment.body == '/oblt-deploy' || github.event.comment.body == '/oblt-deploy-serverless') }}
```

Comment:
```suggestion
    if: ${{ github.event.issue.pull_request && (startsWith(github.event.comment.body, '/oblt-deploy')  || startsWith(github.event.comment.body, '/oblt-deploy-serverless') }}
```

Reply:
`startsWith` open a possibility of parse something we do not want to because of that I have used the exact match.